Example/Controllers/SwiftUI/Features/Settings/MiniAppSettingsView.swift

```diff
@@ -196,9 +183,15 @@ struct MiniAppSettingsView: View {
                     title: MASDKLocale.localize("miniapp.sdk.ios.param.save_title"),
                     message: MASDKLocale.localize("miniapp.sdk.ios.param.save_text")
                 )
-            case let .error(error):
+            case .error:
                 showFullProgress = false
-                alertMessage = MiniAppAlertMessage(title: "Error", message: error.localizedDescription)
+				if viewModel.listConfigI.error != nil && viewModel.listConfigII.error != nil {
+					alertMessage = MiniAppAlertMessage(title: "Error", message: "Something went wrong. Failed to load both lists.")
+				} else if viewModel.listConfigI.error != nil {
+					alertMessage = MiniAppAlertMessage(title: "Error", message: "Something went wrong. Failed to load List I.")
+				} else if viewModel.listConfigII.error != nil {
+					alertMessage = MiniAppAlertMessage(title: "Error", message: "Something went wrong. Failed to load List II.")
+				}
```

Comment:
I see we are trying to add a generic error here. Can we also handle QPS errors and other erros which displayErrorAlert method handles ? 
https://github.com/rakutentech/ios-miniapp/blob/f1705124a7270e52512af5c0353317fd2be13a6e/Example/Controllers/Settings/SettingsTableViewController.swift#L123

Reply:
I'll added the qps error, the code != 200 shouldn't be a concern since it's also just displaying the list couldn't be loaded error